pkg/crc/image/image.go

```diff
@@ -95,39 +112,63 @@ func getImageName(preset crcpreset.Preset) string {
 	}
 }
 
-func PullBundle(preset crcpreset.Preset) error {
+func getPresetNameE(imageName string) (crcpreset.Preset, error) {
+	switch imageName {
+	case "openshift-bundle":
+		return crcpreset.OpenShift, nil
+	case "okd-bundle":
+		return crcpreset.OKD, nil
+	case "podman-bundle":
+		return crcpreset.Podman, nil
+	default:
+		return crcpreset.OpenShift, fmt.Errorf("invalid image name '%s' (Should be openshift-bundle, okd-bundle or podman-bundle)", imageName)
+	}
+}
+func GetPresetName(imageName string) crcpreset.Preset {
+	preset, _ := getPresetNameE(imageName)
+	return preset
+}
+
+func PullBundle(preset crcpreset.Preset, imageURI string) (string, error) {
```

Comment:
this function does not only pull, the bundle, but handles pull, uncompress, verify, etc. The naming is not clear